ui/doc/index.jsx

```diff
@@ -96,6 +97,21 @@ export default class Doc extends React.Component {
     })
   }
 
+  _renderArgumentOptionalLabel(defaultValue) {
+    return <div className='doc-argument_optional'>
+      {defaultValue !== undefined ? `(optional, default=${defaultValue})` : '(optional)'}
```

Comment:
@kossnocorp if `defaulValue` is undefined, it isn't really a default value, in terms of documentation 🙂
It can be either number or string, I think. Don't really know how `jsdoc-parse` works
![screen shot 2015-10-26 at 14 17 19](https://cloud.githubusercontent.com/assets/566319/10724143/ed27ae0a-7be3-11e5-8a7e-2c6ff2604967.png)